R/calculate.R

```diff
@@ -34,27 +34,41 @@ calculate <- function(x, stat, ...) {
   if (stat == "mean") {
     col <- setdiff(names(x), "replicate")
     df_out <- x %>%
+      dplyr::group_by(replicate) %>% 
       dplyr::summarize(stat = mean(!!sym(col)))
   }
 
   if (stat == "prop") {
     col <- attr(x, "response")
     success <- quo(get_par_levels(x)[1])
     df_out <- x %>%
-      dplyr::summarize(stat = mean(!! col == eval_tidy(success))) # This doesn't appear to be working
+     # dplyr::summarize(stat = mean(!! col == rlang::eval_tidy(success))) # This doesn't appear to be working
+      dplyr::group_by(replicate) %>% 
```

Comment:
hmm, weird. whenever i got that error i was assuming it wasn't doing the evaluation correctly. when you call the attributes on the data set at the point, i belive it does show that it is grouping by that...

but hey, if this fixes it, might not be worth figuring out why the first group by isn't taking.